src/main/webapp/WEB-INF/views/UserPageVer1.2.html

```diff
@@ -0,0 +1,373 @@
+                    <b class="caret"></b></a>
+                    <ul class="dropdown-menu">
+                        <li>
+                            <div class="navbar-content">
+                                <div class="row">
+                                    <div class="col-md-5 col-xs-5">
+                                        <img src="http://vev.ru/uploads/images/00/07/64/2011/10/12/c26b1b.jpg"
+                                             alt="AltText" class="img-responsive"
+                                             width="120px" height="120px"/>
+                                    </div>
+                                    <div class="col-md-7 col-xs-7" style="margin-left: -35px">
+                                        <div class="container">
+                                            <div>Welcome back,<strong> UserName!</strong></div>
+                                            <div class="container"
+                                                 style="margin-top:5px; font-size: 15px; font-weight: 500">
+                                                <ul class="non-marker">
+                                                    <li><a href="#"><i class="fa fa-user-o" aria-hidden="true"
+                                                                       style="margin-right: 5px"></i>My account</a></li>
+                                                    <li><a href="#"><i class="fa fa-star" aria-hidden="true"
+                                                                       style="margin-right: 5px"></i>Favourites</a></li>
+                                                    <li><a href="#"><i class="fa fa-question-circle-o"
+                                                                       style="margin-right: 5px"></i>Help</a></li>
+                                                    <li><a href="#" class="btn btn-success btn-md"
+                                                           style="margin-top: 5px"><i class="fa fa-sign-out"
+                                                                                      aria-hidden="true"></i>Sign
+                                                        Out</a></li>
+                                                </ul>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </li>
+                    </ul>
+                </li>
+            </ul>
+        </div>
+    </div>
+</div>
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12">
+            <div class="panel panel-success">
+                <div class="panel-heading">
+                    <h4>User Profile</h4>
+                    <a class="btn btn-edit pull-right" href="EditUserPage.html">
+                        <h4><i class="glyphicon glyphicon-pencil"></i>Edit</h4>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)